templates/workspaces/members/edit.html

```diff
@@ -53,8 +53,8 @@ <h2 class="h3">Manage Access <div class="subtitle">Grant access to an environmen
   </div>
 
   {% for project in projects %}
-  {% set revoke_modal_name = project.name + 'RevokeModal' %}
-  <edit-project-roles inline-template v-bind:name="'{{ project.name }}'" v-bind:id="'{{ project.id }}'">
+  {% set revoke_modal_name = string(project.id) + 'RevokeModal' %}
```

Comment:
I believe you can use the filter directly and then don't need any of the changes in `filters.py`: 
```suggestion
  {% set revoke_modal_name = (project.id|string) + 'RevokeModal' %}
```